openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -60,10 +60,12 @@ static public void loadLibs() {
 
         if (!libsLoaded) {
             final CpuArchitecture cpuArchitecture = getCpuArchitecture(System.getProperty("os.arch"));
+            final boolean isAlpine = isAlpine();
+
             try {
-                loadSharedLibraryFromJar("libgomp.so.1.0.0", cpuArchitecture);
-                loadSharedLibraryFromJar("lib_lightgbm.so", cpuArchitecture);
-                loadSharedLibraryFromJar("lib_lightgbm_swig.so", cpuArchitecture);
+                loadSharedLibraryFromJar("libgomp.so.1.0.0", cpuArchitecture, isAlpine);
```

Comment:
We need to adapt the pipelines in https://github.com/feedzai/feedzai-openml-java/blob/master/.github/workflows/build.yml#L18 to run mvn test in a alpine runner.